src/web/wms/src/main/resources/GeoServerApplication.properties

```diff
@@ -2,6 +2,8 @@
 #pom.xml
 
 AbstractStylePage.confirmOverwrite = Really want to overwrite the current editor contents?
+AbstractStylePage.generate         = Generate ...
+AbstractStylePage.generateNew      = Generate from template style
```

Comment:
Would describe this as "Generate default style" (since from the users perspective they cannot supply templates anywhere).